backend/common/src/main/java/ai/verta/modeldb/common/connections/UAC.java

```diff
@@ -52,9 +54,24 @@ public UAC(
     workspaceServiceFutureStub = WorkspaceServiceGrpc.newFutureStub(authServiceChannel);
     authzServiceFutureStub = AuthzServiceGrpc.newFutureStub(authServiceChannel);
     roleServiceFutureStub = RoleServiceGrpc.newFutureStub(authServiceChannel);
+    serviceAccountRoleServiceFutureStub = RoleServiceGrpc.newFutureStub(authServiceChannel)
+            .withInterceptors(MetadataUtils.newAttachHeadersInterceptor(getServiceUserMetadata(config)));
     organizationServiceFutureStub = OrganizationServiceGrpc.newFutureStub(authServiceChannel);
   }
 
+  private Metadata getServiceUserMetadata(Config config) {
+    Metadata requestHeaders = new Metadata();
+    Metadata.Key<String> email_key = Metadata.Key.of("email", Metadata.ASCII_STRING_MARSHALLER);
+    Metadata.Key<String> dev_key =
+            Metadata.Key.of("developer_key", Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
Set the `developer-key` version too.

Reply:
but we do not have the `developer_key` field in our config file. should i add it??